js/store/reducers/error.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * A reducer that manages the errors for all the actions dispatched.
+ * To work the actions need to use the conventional flux/redux naming:
+ *
+ * - ACTION_NAME_(REQUEST|SUCCESS|FAILURE)
+ *
+ * @flow
+ */
+
+import _ from 'lodash'
+
+import { type GlobalState } from '../../reducers/types'
+import { type Action } from '../../actions/types'
+
+// TODO: Check if is possible to add a better typing for this (don't think so)
+export type ErrorState = Object
+
+export const INITIAL_STATE = {}
+
+/**
+ * Create a selector that return the first error found if any of the actions passed as parameter is in error.
+ *
+ * USAGE: `createErrorSelector(['PROFILE_LOAD', 'PREFERENCES_LOAD'])`
+ */
+export const createErrorSelector = (actions: Array<string>) => (
```

Comment:
you can use a `$ReadOnlyArray<string>` here (this is true most times you work with arrays and/or accept arrays as parameter)